src/Dfc.CourseDirectory.WebV2/Features/HelpdeskDashboard/Dashboard.cshtml

```diff
@@ -3,18 +3,21 @@
 }
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-        <div class="govuk-grid-row">
-            <div class="govuk-grid-column-one-half">
-                <div class="pttcd-helpdesk-dashboard-card">
-                    <h2 class="govuk-heading-m">Quality assurance</h2>
-                    <p>New provider and apprenticeship training courses.</p>
-                    <ul class="govuk-list">
-                        <li><a asp-controller="ApprenticeshipQA" asp-action="ListProviders" class="govuk-link">QA submissions</a></li>
-                        <li><a href="#" class="govuk-link">Download QA report</a></li>
-                    </ul>
+    @if (Model.CanAccessApprenticeshipQA)
```

Comment:
`if` not needed here (suspect a bad merge)

Reply:
done